src/tanstack/react/index.ts

```
@@ -0,0 +1,243 @@
+  }
+
+  // For nested routers, ensure methods are non-enumerable so they don't get lost in serialization
+  // This is a defensive measure to ensure methods are preserved
+  const makeMethodsNonEnumerable = (obj: Record<string, unknown>) => {
+    if (obj && typeof obj === 'object') {
+      for (const key in obj) {
+        const value = obj[key] as unknown;
+        if (value && typeof value === 'object') {
+          // Check if it's an endpoint-like object
+          const endpointLike = value as {
+            execute?: Function;
```

Biome (1.9.4)

[error] Don't use 'Function' as a type.

Prefer explicitly define the function shape. This type accepts any function-like value, which can be a common source of bugs.

(lint/complexity/noBannedTypes)